### ISSUE TITLE
Uikit 3.1.0 fails to build with node sass

### ISSUE THREAD

**Issue by shawjak3** [uikit/uikit#3832]
### UIkit version

3.1.0

### Browser
Chrome

### Steps to reproduce
Newest version fails to build with webpack. Not sure if this is your issue or node sass so please let me know if this needs to be posted there as well.

### What is Expected?
The scss should build.

### What is actually happening?
```Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/sass-loader/lib/loader.js):
.uk-heading-primary when ($deprecated = true) {
                        ^
      Invalid CSS after "...ng-primary when": expected selector, was "($deprecated = true"
      in /app/node_modules/uikit/src/scss/components/heading.scss (line 125, column 26)
    at runLoaders (/app/node_modules/webpack/lib/NormalModule.js:301:20)
    at /app/node_modules/loader-runner/lib/LoaderRunner.js:367:11
    at /app/node_modules/loader-runner/lib/LoaderRunner.js:233:18
    at context.callback (/app/node_modules/loader-runner/lib/LoaderRunner.js:111:13)
    at Object.render [as callback] (/app/node_modules/sass-loader/lib/loader.js:52:13)
    at Object.done [as callback] (/app/node_modules/neo-async/async.js:8077:18)
    at options.error (/app/node_modules/node-sass/lib/index.js:294:32)
 ERROR  Build failed with errors.```

**Comment by happilymarrieddad** [comment]
This issue is happening for me as well

**Comment by arminmon** [comment]
Same here.
```
Libsass: Error: Invalid CSS after "...ng-primary when": expected selector, was "($deprecated = true"
        on line 125 of Users/arminmon/Dev/web-design/TripGel/node_modules/uikit/src/scss/components/heading.scss
        from line 8 of Users/arminmon/Dev/web-design/TripGel/node_modules/uikit/src/scss/components/_import.scss
        from line 7 of Users/arminmon/Dev/web-design/TripGel/node_modules/uikit/src/scss/uikit-theme.scss
        from line 5 of Users/arminmon/Dev/web-design/TripGel/source/styles/main.scss
>> .uk-heading-primary when ($deprecated = true)
```

**Comment by arminmon** [comment]
Same here.

**Comment by SaphiLC** [comment]
same here -_-

**Comment by mohamedhamad** [comment]
same here

**Comment by mohamedhamad** [comment]
looks like the syntax in the /node_modules/uikit/src/scss/components/heading.scss should be 
**.uk-heading-hero:when($deprecated)**
not
**_.uk-heading-hero when ( $deprecated=true )_**

**Comment by dalu** [comment]
I replaced `when ($deprecated = true)` with empty space `` which made the problem go away

**Comment by janschoenherr** [comment]
Thanks, this should be resolved with 3.1.1
